.github/workflows/ci.yaml

```diff
@@ -0,0 +1,59 @@
+name: bloom-ci
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['*']
+
+jobs:
+    build:
+      strategy:
+        matrix:
+          os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, macos-latest]
+          python: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+          exclude:
+          - os: ubuntu-16.04
+            python: 3.6
+          - os: ubuntu-16.04
+            python: 3.7
+          - os: ubuntu-16.04
+            python: 3.8
+          - os: ubuntu-16.04
+            python: 3.9
+          - os: ubuntu-18.04
+            python: 3.5
+          - os: ubuntu-20.04
+            python: 2.7
+          - os: ubuntu-20.04
+            python: 3.5
+          - os: ubuntu-20.04
+            python: 3.6
+          - os: ubuntu-20.04
+            python: 3.7
+          - os: macos-latest
+            python: 3.5
+          - os: macos-latest
+            python: 3.6
+      name: bloom tests
+      runs-on: ${{matrix.os}}
+
+      steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{matrix.python}}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{matrix.python}}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools
+          python -m pip install PyYAML argparse empy rosdep vcstools catkin-pkg python-dateutil
+          python -m pip install nose coverage pep8
```

Comment:
If any of these are already installed as system packages, they might not get upgraded, right? Since we're using pip, I'm assuming that we're trying to test against the latest versions of these packages, and not the system packages.

That said, I can see that this is the same behavior as was used in the Travis config, so I'm OK with moving forward without discussing it for now in an effort to maintain parity in this PR.

Reply:
To ensure update to the latest versions out of pip we'd add the `-U` flag right?

I would be fine making that change here if you think it's a good idea. I haven't given it any thought.